src/webapp/components/template-selector/TemplateSelector.tsx

```diff
@@ -289,88 +293,112 @@ export const TemplateSelector = ({
 
     return (
         <>
-            <h3 className={classes.title}>{i18n.t("Template")}</h3>
-
-            <div className={classes.row}>
-                {models.length > 1 && (
+            <Section
+                title={<h3 className={classes.title}>{i18n.t("Template")}</h3>}
+                classProps={{ section: classes.section }}
```

Comment:
[immutable props] I think we can simply pass "classes" here (unless we really don't want the header). If that's the case, we can use useMemo to build the object. In this case, we don't use useMemo because it's costy operation, but to keep the value immutable.